.github/workflows/static-code-analysis.yaml

```diff
@@ -0,0 +1,16 @@
+name: Static code analysis
+
+on: [pull_request]
+
+jobs:
+    static-code-analysis:
+        strategy:
+            matrix:
+                php: [7.1, 7.2, 7.3, 7.4]
```

Comment:
no need to run phpstan 4 times. The result will be the same each time AFAIK.

Reply:
I actually did have additional errors while running PHPStan with the PHP7.4 image.

@jakzal did mention changing the image to be running `composer install` in https://github.com/php-translation/symfony-storage/pull/38#discussion_r356660383

Will revert the matrix for now 👍

Reply:
Were the different images using the same version of PHPStan ?